node_modules/oae-core/topnavigation/topnavigation.html

```diff
@@ -13,7 +13,7 @@
         <!-- RIGHT MENU -->
         <div id="topnavigation-right" class="col-xs-5 text-right">
             <!-- SEARCH FORM -->
-            <form id="topnavigation-search-form" class="form-inline" role="search" aria-label="__MSG__TOPNAV_SEARCH_ARIA_LABEL__">
+            <form action="." id="topnavigation-search-form" class="form-inline" role="search" aria-label="__MSG__TOPNAV_SEARCH_ARIA_LABEL__">
```

Comment:
@sathomas : I believe there are more search forms that would need to be adjusted (e.g. library searches, admin UI, etc.).